src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilderRow.tsx

```diff
@@ -39,61 +46,95 @@ const QueryBuilderRow: React.FC<IQueryBuilderRowProps> = (props) => {
         onChangeValue,
         position,
         removeRow,
-        styles
+        styles,
+        theme,
+        updateSnippet
     } = props;
+    const propertySelectorId = useId('cb-advanced-search-property-select');
```

Comment:
can this just be a const at module scope?

Reply:
It seems like this is creating a unique id for every row with the string provided as a prefix. I'll leave it like this because it is required for the callout to target the correct react-select dropdown.

Reply:
neat. good to know. thanks.